NineChronicles.Standalone.Tests/GraphTypes/StandaloneMutationTest.cs

```diff
@@ -26,8 +27,15 @@ namespace NineChronicles.Standalone.Tests.GraphTypes
 {
     public class StandaloneMutationTest : GraphQLTestBase
     {
+        private readonly Dictionary<string, string> _sheets = null;
+
+        private readonly TableSheets _tableSheets = null;
+
         public StandaloneMutationTest(ITestOutputHelper output) : base(output)
         {
+            var fixturePath = Path.Combine("..", "..", "..", "..", "Lib9c", ".Lib9c.Tests", "Data", "TableCSV");
```

Comment:
This path seems not referencing the submodule.

Reply:
Isn't it referencing correctly? The CI test was successful.

Reply:
Oh, I thought test's working directory would be same with this file.